bindings/pyroot/pythonizations/test/tf_pycallables.py

```diff
@@ -94,6 +97,36 @@ def test_fitgauss(self):
         self.assertAlmostEqual(mean, 0.0, 2)
         self.assertAlmostEqual(abs(std), 1.0, 2)
 
+    def test_evalpar(self):
+        """
+        Test the 2D Numpy array pythonisations for TF1::EvalPar
+        """
+        import numpy as np
+
+        rtf1_coulomb = ROOT.TF1("my_func", pyf_tf1_coulomb, -10, 10, ndim=2)
```

Comment:
Can we also have another test that does not pass the `ndim` parameter, to exercise the improvement you introduced?